RecoTracker/IterativeTracking/python/InitialStep_cff.py

```diff
@@ -315,13 +316,14 @@
      qualityCuts = [-0.95,-0.85,-0.75]
 ))
 
-from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
-from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-trackdnn.toReplaceWith(initialStep, TrackLwtnnClassifier.clone(
-        src         = 'initialStepTracks',
-        qualityCuts = [0.0, 0.3, 0.6]
+from RecoTracker.FinalTrackSelectors.TrackTfClassifier_cfi import *
+from RecoTracker.FinalTrackSelectors.trackSelectionTf_cfi import *
+trackdnn.toReplaceWith(initialStep, TrackTfClassifier.clone(
+        src = 'initialStepTracks',
+        qualityCuts = qualityCutDictionary["InitialStep"]
 ))
-(trackdnn & fastSim).toModify(initialStep,vertices = 'firstStepPrimaryVerticesBeforeMixing')
```

Comment:
this change with the whitespace and quotes is not strictly needed, please revert